beaker.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'minitar', '~> 0.5.4'
-  s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'json', '~> 1.4'
```

Comment:
Maybe, but I kind of like it being at least a bit explicit. I would almost recommend `~> 1`, just to catch the upgrade to json 2.x